util/pace/util/grid/generation.py

```diff
@@ -222,8 +222,11 @@ def __init__(
         quantity_factory: util.QuantityFactory,
         communicator: util.CubedSphereCommunicator,
         grid_type: int = 0,
+        dx_const: float = 1000.0,
+        dy_const: float = 1000.0,
+        deglat: float = 15.0,
     ):
-        assert grid_type < 3
+        # assert grid_type < 3
```

Comment:
Assert or assert not, there's no try.

Reply:
That'll get taken out when we put the grid config in, though since it hasn't been implemented yet maybe we should keep the assert in?

Reply:
I'd say keep it in.